tools/packaging/kernel/configs/fragments/common/fs.conf

```diff
@@ -31,7 +31,6 @@ CONFIG_FSNOTIFY=y
 CONFIG_DNOTIFY=y
 CONFIG_INOTIFY_USER=y
 CONFIG_FANOTIFY=y
-CONFIG_AUTOFS4_FS=y
```

Comment:
Nit: it seems CONFIG_AUTOFS4_FS should be added to the commit changelog along with the other ones for consistency.